test/extended/cli/mustgather.go

```diff
@@ -0,0 +1,64 @@
+package cli
+
+import (
+	g "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
```

Comment:
Please stick to the import organization we have, which is:
1. builtin imports
2. other imports
3. k8s imports
4. origin imports.

Reply:
Also wrt gomega we try to have an alias `o` is sufficient rather than importing everything into current namespace.